.github/ISSUE_TEMPLATE/20-bug-report.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Bug report
+about: Report a mistake or inconsistency in the specification code or documentation text.
```

Comment:
Would people use the same template to report a bug in the TCK? Should we add that to the description?

Reply:
ok

Reply:
done